desktop_source/main.cpp

```diff
@@ -90,6 +91,7 @@ static gboolean _user_input_handler (
 
 int main (int argc, char *argv[])
 {
+    InitWFDLogging();
```

Comment:
Please no. Don't force the apps to call magic initialization functions just to have logging.The linking between our logging functions and the actual logging facility provided by the OS should happen at compile time.